README.md

```diff
@@ -34,3 +34,27 @@ Our OpenBioML ChemNLP project is not afiliated to the [ChemNLP library from NIST
 
 # About OpenBioML.org
 See https://openbioml.org, especially [our approach and partners](https://openbioml.org/approach-and-partners.html).
+
+# Installation and set-up
+Create a new conda environment for Python 3.8:
+```
+conda create -n chemnlp python=3.8
+conda activate chemnlp
```

Comment:
should we recommend installing the env via our `yml` file? Might simplify if some dependencies (cudatoolkit, etc.) are only available via conda.

Reply:
Sure, though we'll need to be clear on options e.g. is the `chemnlp` package installed or not.